README.md

```diff
@@ -94,7 +94,8 @@ There are numerous ways to install and configure a complete Kubernetes cluster.
 For reference material, we have provided existing Kubernetes deployment models that have been verified to work with CORTX on Kubernetes. These are only provided for reference and are not meant to be explicit deployment constraints.
 
 1.  [Seagate Internal Jenkins Job](http://eos-jenkins.mero.colo.seagate.com/job/Cortx-kubernetes/job/setup-kubernetes-cluster/)
-2.  [CORTX on AWS and Kubernetes - Quick Install Guide](https://github.com/Seagate/cortx-k8s/blob/main/doc/cortx-aws-k8s-installation.md)
+2.  [CORTX on AWS and Kubernetes - Quick Install Guide](https://github.com/Seagate/cortx-k8s/blob/UDX-6683_move_documentation_to_readme_md/doc/cortx-aws-k8s-installation.md)
```

Comment:
In merging your changes here, you clobbered a recent update I made to this page. The URL for this link should be https://github.com/Seagate/cortx-k8s/blob/main/doc/cortx-aws-k8s-installation.md (per https://github.com/Seagate/cortx-k8s/commit/860ef5b6054ddbb100256c015f49ce2cd03dc90d).

@alfhad Can you update this specific link to point to `main` instead of the now-deleted branch?

Your link below looks good.

Reply:
Done!